RetrospectiveExtension.Frontend/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Development version to publish dev version of extension
+vss-extension-dev.json
```

Comment:
do we need this nested `.gitignore`? If so, the commented out one should be removed from the main one. But i'm not sure we need a separate one for the frontend.

Reply:
will leave everything in the top level .gitignore since many folks left similar comments